cmake/partition_manager.cmake

```diff
@@ -312,14 +312,21 @@ set(pm_cmd
   ${region_arguments}
   )
 
+
+set(DTS_ROOT_BINDINGS ${CACHED_DTS_ROOT_BINDINGS})
+
 set(pm_output_cmd
   ${PYTHON_EXECUTABLE}
   ${ZEPHYR_NRF_MODULE_DIR}/scripts/partition_manager_output.py
   --input-partitions ${pm_out_partition_file}
   --input-regions ${pm_out_region_file}
   --config-file ${pm_out_dotconf_file}
+  --dts-file ${ZEPHYR_DTS}
```

Comment:
seems like this falls apart if someone has an overlay?

Reply:
Okay please explain. I thought the overlay was applied to the final DTS `zephyr.dts` but please correct my interpretation of the system.

Reply:
https://docs.zephyrproject.org/latest/_images/build-config-phase.svg

From this diagram it seems that they are combined and finally outputted into a Zephyr.dts file.